client/src/components/Companies/Scores.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from "react";
+import "../../styles/Companies.css";
+import DiversityImg from "../../assets/diversity.png";
+import WorkerExploitImg from "../../assets/workerexploit.png";
+import WasteImg from "../../assets/wastepollution.png";
+import SustainableImg from "../../assets/sustainable.png";
+
```

Comment:
the big score component should consist of 4 same type small component.   
Please define a sub-component and use it to construct the big component

Reply:
Oh so like a component for  A score, Bscore , C score and D score?

Reply:
No, one single component for A, B, C and D, because the structures of them are the same.
It should use parameters to render different content.